app/editor/src/features/content/validation/ContentFormSchema.ts

```diff
@@ -4,37 +4,41 @@ import { array, date, number, object, string } from 'yup';
 export const ContentFormSchema = object().shape(
   {
     productId: number().required('Product is required.').notOneOf([0], 'Product is required.'),
-    sourceId: number().when('tempSource', {
-      is: undefined,
-      then: number().required('Either source or other source is required.'),
+    sourceId: number().when('tempSource', (value: any) => {
```

Comment:
Updated to resolve `yup` breaking changes.  Rolled back `yup` however due to the number of breaking changes.  Kept this file update since it works and will be supported when we upgrade.